packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,13 +5,13 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.5
+    tag: v1.0.6-rc1
```

Comment:
We'll need to remember to cut non rc version, you will get reminder as part of release.